code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -66,1037 +66,1240 @@ Follow by example and make good judgement based on length which list to include
 		icon_state = "hair_dreads"
 		length = 4
-		chatname = "Dreadlocks"
-
-	curls
-		name = "Curls"
-		icon_state = "hair_curls"
-		name = "Curled Hair"
-
-	afro
-		name = "Afro"
-		icon_state = "hair_afro"
-		length = 4
-		chatname = "Afro"
-
-	afro2
-		name = "Afro 2"
-		icon_state = "hair_afro2"
-		length = 4
-		chatname = "Afro"
+		chatname = "dreadlocks"
 
-	afro_large
-		name = "Big Afro"
-		icon_state = "hair_bigafro"
-		gender = MALE
-		length = 4
-		chatname = "Big Afro"
-
-	rows
-		name = "Rows"
-		icon_state = "hair_rows1"
-		length = 2
-		chatname = "Cornrows"
+	drills_celes
+		name = "Drills, Celes"
+		icon_state = "hair_drills_celes"
+		length = 3
+		chatname = "drilled hair"
 
-	rows2
-		name = "Rows 2"
-		icon_state = "hair_rows2"
+	drills_drillruru
+		name = "Drills, Drillruru"
+		icon_state = "hair_drills_drillruru"
+		gender = FEMALE
 		length = 2
-		chatname = "Cornrows"
-
-	sargeant
-		name = "Flat Top"
-		icon_state = "hair_sargeant"
-		gender = MALE
-		chatname = "Short Hair"
-		length = 0
+		chatname = "drilled hair"
 
 	emo
 		name = "Emo"
 		icon_state = "hair_emo"
-		chatname = "Short Hair"
-		length = 0
+		chatname = "fringed hair"
 
-	longemo
-		name = "Long Emo"
-		icon_state = "hair_emolong"
-		gender = FEMALE
+	emo_alt
+		name = "Emo, Alt"
+		icon_state = "hair_emo_alt"
+		chatname = "fringed hair"
+
+	emo_long
+		name = "Emo, Long"
+		icon_state = "hair_emo_long"
+		chatname = "long fringed hair"
 		length = 2
-		chatname = "Short Hair"
 
-	fringeemo
-		name = "Emo Fringe"
+	emofringe
+		name = "Emo, Fringe"
 		icon_state = "hair_emofringe"
-		chatname = "Fringed Hair"
-		length = 1
+		chatname = "fringed hair"
 
-	shortovereye
-		name = "Overeye Short"
-		icon_state = "hair_shortovereye"
-		chatname = "Short Hair"
-		length = 0
+	emofringe_long
+		name = "Emo, Fringe Long"
+		icon_state = "hair_emofringe_long"
+		gender = FEMALE
+		length = 3
+		chatname = "long fringed hair"
 
-	veryshortovereyealternate
-		name = "Overeye Very Short, Alternate"
-		icon_state = "hair_veryshortovereye"
-		chatname = "Short Hair"
-		length = 0
+	fade_high
+		name = "Fade, High"
+		icon_state = "hair_fade_high"
+		gender = MALE
+		chatname = "unshaved hair"
 
-	longovereye
-		name = "Overeye Long"
-		icon_state = "hair_longovereye"
-		length = 2
-		chatname = "Long Hair"
+	fade_low
+		name = "Fade, Low"
+		icon_state = "hair_fade_low"	//why did we have to pick from a letter before M for something low
+		gender = MALE
+		chatname = "short hair"
 
-	fag
-		name = "Flow Hair"
-		icon_state = "hair_f"
+	fade_medium
+		name = "Fade, Medium"
+		icon_state = "hair_fade_medium"
+		gender = MALE
+		chatname = "unshaved hair"
+
+	fade_none
+		name = "Fade, None"
+		icon_state = "hair_fade_none"
+		gender = MALE
+		chatname = "short hair"
+
+	familyman
+		name = "Family Man, The"
+		icon_state = "hair_thefamilyman_the"
+		gender = MALE
+		chatname = "homely hairdo"
 		length = 2
-		chatname = "Flowing Hair"
+
+	father
+		name = "Father"

```

Comment:
YMMV and all
but I'm _pretty sure_ this pales when compared to other things like Joestar.

Reply:
Then why not remove Joestar?

Reply:
Then why not remove Joestar?

Reply:
I know I'm going to sound like a pretentious bohemian hippie but hear me out:

Because the two of us alone (plus whoever might be going through these) might not be able to think of something creative with them where someone else will.

Chances are, people are going to overlook these anyways; but for those who might actually have something in mind (e.g., an ancient wise wizard), it'll be there. Sure, maybe you might get some "greytide baldie griefer" who comes in rocking something, but that's really a problem in itself already before we even talk about how they look — one of which gets dealt with by staff _anyways._
That's also noting that you can make "dumb appearances" with what we already have. And even still that doesn't necessarily guarantee an uninteresting/"snowflake"/"baldie" character/player if it can be justified.

I imagine they would look quite all right too with the right facial hair, the right clothes, and the right characterisation in mind for a narrative.

Reply:
"They'll overlook these anyways" and "we're limiting potential" is a bad argument. Our hairstyle selection is already super bloated. Bad sprites shouldn't be readded.

Reply:
Not to sound scorned but I'm in the belief that writing something off as a "bad argument" seemingly without actually addressing them isn't quite so strong of an argument in itself because that doesn't seem like a discussion can go anywhere if no-one ever actually takes an alternative perspective and clings to their own. Taking note that this also probably a grey area in which no one side or opinion is some sort of universal truth that all people should adhere to.

My point again: no individual person, whether that's either of us or someone else reading these, no matter how right we think we are, it would probably be healthier for everyone if we didn't cling so tightly to something without ever thinking about it.

Regarding bloat, if I'm interpreting it as file size, I'm under the impression that they aren't actually taking up as much space as people constantly hail them to. That's notwithstanding the fact that we also went for massive additions in audio with a large array of .oggs which take up far, far more space than any file that tells a system which pixel to light up.

I genuinely appreciate that you've taken the time to read through these. But I took the time to make them. True, that doesn't necessarily give me the right to get this pushed through. True, others have worked on "objectively" harder things. But I would really appreciate if it could at least be considered, even if I'll try not to mention that otherwise.

So I ask if you, and/or anyone else who would like to say so, could please explain to me why you feel those are bad arguments, and correct me if I may be misunderstood with bloating. I wish to understand.

Reply:
The hair menu is hard to traverse, and find things in, because of how many things are there.